models/Tag.js

```diff
@@ -0,0 +1,31 @@
+import slugify from 'slugify';
+
+export default (sequelize, DataTypes) => {
+  const Tag = sequelize.define('Tag', {
+    tagName: {
+      type: DataTypes.STRING,
+      primaryKey: true,
```

Comment:
#### Nitpick
- Isn't it enough to have `tagNameId` as the primary key?

Reply:
The reason for that is because the joined table `ArticleTag` model makes reference to it.

Writing about it just gave me an idea. First of all, I'll do away with having the tagName as a `primaryKey`. Thanks.